testing/rust/musl-fix-static-linking.patch

```diff
@@ -0,0 +1,148 @@
+--- a/src/librustc/session/mod.rs 
++++ b/src/librustc/session/mod.rs
+@@ -409,6 +409,11 @@
+         return crt_static;
+     }
+ 
++    pub fn fully_static(&self) -> bool {
```

Comment:
What exactly does this mean? Static without PIE? This patch also miss description. :)